fifa_ratings_predictor/crawler/crawler/settings.py

```diff
@@ -23,7 +25,7 @@
 ROBOTSTXT_OBEY = True
 
 # Configure maximum concurrent requests performed by Scrapy (default: 16)
-CONCURRENT_REQUESTS = 4
+CONCURRENT_REQUESTS = 16 # before: 4
```

Comment:
Seems a bit heavy? I think setting it privately to do that is fair, but allowing everyone to hit the website so heavily so easily seems a bit harsh

Reply:
The default is set to 16 and per default the maximum per domain is 8. I think using the default is fine, but it can also stay at 4 and be changed privately if necessary ;)